docs/concepts/feature-registry.md

```diff
@@ -74,6 +74,8 @@ client.register_features()
 all_features = client.list_registered_features(project_name=client.project_name)
 ```
 
+Please avoid applying a same name to different features under a certain project. Since it will be treated as updating an exsiting project which is not supported by feathr and will cause errors.
```

Comment:
Should this be done in validation instead of add a no do in doc? Probably a feature request should be opened to add a validation in registry side

Reply:
I think we just added this validation by this commit: https://github.com/feathr-ai/feathr/commit/d59ea4bdad6606683470883e686a56262c5f10b3